packages/api-derive/src/balances/votingBalances.ts

```diff
@@ -11,9 +11,9 @@ import { drr } from '../util/drr';
 import { votingBalance } from './votingBalance';
 
 export function votingBalances (api: ApiInterface$Rx) {
-  return (...params: Array<any>): Observable<Array<DerivedBalances>> => {
-    const addresses: Array<AccountIndex | AccountId | string> = params.slice(0, params.length - 1);
-
-    return combineLatest(...addresses.map(votingBalance(api))).pipe(drr());
+  return (addresses?: Array<AccountIndex | AccountId | string>): Observable<Array<DerivedBalances>> => {
```

Comment:
Here we had a small slice bug - just due to the differences between the comnbineLatest.